service/siteService.js

```diff
@@ -17,27 +17,31 @@ module.exports = {
 		const cachePath = cacheService.getCachePath(requestPath);
 		const cachedResult = cacheService.readCache(cachePath);
 
-		if (cachedResult != null) {
-			console.log("Cache hit. Returning cached result for " + requestPath);
-			return cachedResult
-		} else {
-			const siteRecords = await airtableClient.getMutualAidSites()
-				.catch( e => {
-					console.error("There was an error getting mutual aid sites: " + e.message)
-					// TODO: Send slack? alert so there's visibility into the error!!
-					return cacheService.readCacheBypassInterval(cachePath)
-				})
-			const result = siteRecords
-					.filter(validateRecord)
-					.map(mapRecordFields)
-			cacheService.writeCache(cachePath, result);
-			return result
+		try {
+			if (cachedResult != null) {
+				console.log("Cache hit. Returning cached result for " + requestPath);
+				return cachedResult
+			} else {
+				const siteRecords = await airtableClient.getMutualAidSites()
```

Comment:
Is this guaranteed to always return a non-null value?

Reply:
mmmm yes? I think? non-null or it will throw an error I think? Although I guess I'm not 100% sure what the airtable library returns if it finds no records...like if it returns empty object or if it's undefined....